app/src/main/res/layout/loyalty_card_view_layout.xml

```diff
@@ -171,20 +185,80 @@
                 app:layout_constraintTop_toBottomOf="@+id/minimizeButton"
                 tools:visibility="visible" />
 
-            <SeekBar
-                android:id="@+id/barcodeScaler"
+            <LinearLayout
+                android:id="@+id/widthScalerLayout"
+                app:layout_constraintBottom_toBottomOf="@+id/heightScalerLayout"
+                app:layout_constraintEnd_toEndOf="parent"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintTop_toTopOf="@+id/widthScalerGuideline"
                 android:layout_width="match_parent"
                 android:layout_height="wrap_content"
-                android:layout_margin="@dimen/inputPadding"
-                android:layout_marginStart="15.0dip"
-                android:layout_marginEnd="15.0dip"
-                android:contentDescription="@string/set_scale"
-                android:max="100"
                 android:visibility="gone"
+                android:gravity="center"
+                android:orientation="horizontal" >
+
+                <TextView
+                    android:id="@+id/zoomWidthText"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:layout_marginStart="15.0dip"
+                    android:layout_marginEnd="4.8dip"
+                    android:visibility="gone"
+                    android:gravity="center|left"
+                    />
+
+                <SeekBar
+                    android:id="@+id/barcodeWidthScaler"
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:layout_margin="@dimen/inputPadding"
+                    android:layout_marginEnd="15.0dip"
+                    android:contentDescription="@string/set_scale"
+                    android:max="100"
+                    android:visibility="gone"
+                    app:layout_constraintBottom_toBottomOf="@+id/barcodeScaler"
+                    app:layout_constraintEnd_toEndOf="parent"
+                    app:layout_constraintStart_toStartOf="@+id/verticalGuideline"
+                    app:layout_constraintTop_toTopOf="@+id/scalerGuideline" />
```

Comment:
The width bar for some reason misses the colour that the height bar has:
![image](https://user-images.githubusercontent.com/1885159/197838850-ddc063dd-fcea-4463-92aa-ac00646aaea5.png)

Reply:
I missed that we needed to change the color in the LoyaltyCardViewActivity class. It should match the color of the overall theme now :)